OpenRA.Mods.Common/Scripting/ScriptTriggers.cs

```diff
@@ -421,6 +422,48 @@ public void OnDiscovered(Actor self, Player discoverer, bool playNotification)
 			}
 		}
 
```

Comment:
I was mistaken.
You cannot explicitly implement interface members on `TriggerGlobal`.